images/ubuntu/templates/ubuntu-22.04.pkr.hcl

```
@@ -414,8 +541,9 @@ build {
   }
 
   provisioner "shell" {
+    environment_vars = ["SOURCE_TYPE=${source.type}"]
```

What about using the `only` parameter?

```suggestion
    only                        = ["azure-arm.build_image"]
```

---

fixed